linera-views/src/map_view.rs

```diff
@@ -713,22 +710,26 @@ where
     /// # let context = create_memory_context();
     ///   let mut map = ByteMapView::load(context).await.unwrap();
     ///   map.insert(vec![0,1], String::from("Hello"));
-    ///   assert_eq!(map.get_mut_or_default(vec![7]).await.unwrap(), "");
-    ///   let value = map.get_mut_or_default(vec![0,1]).await.unwrap();
+    ///   assert_eq!(map.get_mut_or_default(&[7]).await.unwrap(), "");
+    ///   let value = map.get_mut_or_default(&[0,1]).await.unwrap();
     ///   assert_eq!(*value, String::from("Hello"));
     ///   *value = String::from("Hola");
     ///   assert_eq!(map.get(&[0,1]).await.unwrap(), Some(String::from("Hola")));
     /// # })
     /// ```
-    pub async fn get_mut_or_default(&mut self, short_key: Vec<u8>) -> Result<&mut V, ViewError> {
+    pub async fn get_mut_or_default(&mut self, short_key: &[u8]) -> Result<&mut V, ViewError> {
```

Comment:
nit: `get_mut` and `get_mut_or_default` are written differently which is a little weird. Apparently:
* `get_mut` is ok accessing `self.updates` several times
* `get_mut_or_default` tries not to do that

Reply:
It was possible indeed to correct that.